scripts/load-versions.js

```diff
@@ -5,7 +5,16 @@
 const fs = require('fs')
 const nodeVersionData = require('node-version-data')
 
-module.exports = nodeVersionData
+module.exports = function (callback) {
+  nodeVersionData((err, versions) => {
+    if (err) {
+      return callback(err)
+    }
+
+    versions.forEach((v) => v.lts === false && delete v.lts)
```

Comment:
sorry, missed it, have updated to use that instead, tbh I didn't know that was possible although I did a quick google for such syntax!